sos/report/plugins/fail2ban.py

```diff
@@ -0,0 +1,40 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, DebianPlugin, UbuntuPlugin
+
+
+class Fail2Ban(Plugin, DebianPlugin, UbuntuPlugin):
```

Comment:
We can make this `IndependentPlugin`  so that this plugin is enabled for RHEL-family distros as well. Taking a quick look, the collection patterns would be the same, just an additional package name would be needed in the `packages` tuple.

Reply:
Thanks for the quick feedback @TurboTurtle ! I updated this part as requested.